src/index.js

```diff
@@ -5,7 +5,19 @@
  */
 
 const hammingDistance = (a, b) => {
-    
+  if (a.length !== b.length) {
+    throw new Error('The length of the strings should be the same.')
   }
-  
-  module.exports = hammingDistance;
\ No newline at end of file
+
+  const distance = a.split('').filter((letter, index) => {
+    if (b[index] !== letter) {
+      return true
+    }
+    return false
+  })
```

Comment:
No es sobreingeniería podiendo resolverlo con un ciclo for sin tener que pasar el string a un array?